src/common/__test__/string-builder.spec.js

```diff
@@ -58,6 +58,14 @@ test('makeParams should add params', () => {
     expect(result).toContain(JSON.stringify(params));
 });
 
+test('makeParams should add params URLSearchParams', () => {
+    const sb = new StringBuilder({ params: true });
+    const params = new URLSearchParams({ param1: 'value1', param2: 'value2' });
```

Comment:
In the following, the variable name data is used, so it would be good to match it here for consistency.
```suggestion
    const data = new URLSearchParams({ param1: 'value1', param2: 'value2' });
```

Reply:
It's a minor detail, but it would be better to match the declaration order with the test code below for consistency.